app/src/main/java/com/example/cocktails/CocktailItemAdapter.kt

```diff
@@ -67,7 +67,7 @@ class CocktailItemAdapter internal constructor(context: Context, data: ArrayList
         val favoriteList = (cnt.applicationContext as Cocktails).mFavorites
         when (toSearch) {
             "favorites" -> items.filter { favoriteList.getBoolean(it.name, false)}.forEach { resultList.add(it) }
-            "custom" -> items.filter { it.isCustom }.forEach { resultList.add(it) }
+            "custom" -> items.filter { it.customIs }.forEach { resultList.add(it) }
```

Comment:
if 'isCustom' cause problems, i think that 'custom' is better than 'customIs'. same for isReview. WDYT?

Reply:
For me its not matter . But I this that "is" represent boolean var, so its much better